content/post/My-journey-with-ALiAS.md

```diff
@@ -0,0 +1,55 @@
+
+Finally, in my fourth semester, I got to meet and be a part of this awesome clan of FOSS patrons: ALiAS.
+
+## It all started with a commit ...
+January 8, 2017.
+
+I was just lying in my room going through all my FB, Instagram and WhatsApp updates when I saw an invite link to the then-called 'ASET ALiAS group' (the first WhatsApp group of ALiAS). At first, I discarded the message as I thought of it to be another entrant in the list of not-so-enthusiastic tech clubs. But later that day, I casually went through its website and found something that lacked in other clubs - a GitHub repo of their website with well-defined contribution guidelines. So that was about some serious start.
+
+I went through the repo and found a layout bug in a webpage's CSS code. Thought of it as a nice & small start, and I did the usual - fork, clone, fix, commit, and push. That single, tiny commit landed me into the first batch of ALiAS members and their first ever group meet. Little I knew, how a single commit was gonna change the course of my next two years of college.
+
+## ... And behold the community!!
+Starting from weekly meetups to some in-house workshops, ALiAS opened up the gates for all of us and people coming after, towards some of the biggest FOSS communities in Delhi-NCR like PyDelhi, ILUG-D, etc. We called people from those communities to our campus for talks, workshops, etc. We came up with our projects for the club newbies to get their hands dirty on, and also tried to organize a hackathon (which didn't go well though). But all that proved to be a large addition to our fun and learning.
+
+And if that wasn't enough, getting along with the fellow ALiASians to various FOSS meetups, devsprints, hackathons was another level of experience. Victory for us wasn't about winning in the events, but leaving our impression on people around us. And I can say, we did win in that case.
+
+So going from a small club revived after years, to a huge community of people like me, the end of which can't (and shouldn't) be defined, ALiAS helped me to bring a new spark in my life. And I can say that no one or nothing can put out this spark, ever.
+
+## FOSS Meetups, projects, hackathons, and a lot more...
+ALiAS gave me a big motivation to get into various FOSS meetups, hackathons, etc. and get involved in their community projects too. Along with learning and experience, all of those connected me to a wider band of people from different regions and professions, and like always the fun moments made to the top. Some honourable mentions among those are:
+
+- Web projects of ALiAS, including its official website.
+- Community meetups of PyDelhi, ILUG-D, LinuxChix India etc.
+
+{{< figure src="/blog/images/megameet.jpg" caption="With my ALiAS mates at FOSS mega meetup at Adobe, Noida" >}}
+{{< figure src="/blog/images/ILUGDmeet.jpg" caption="Recent ILUG-D meetup at The Office Pass, Gurugram" >}}
```

Comment:
Please follow the markdown formatting rules for adding images. Also, we would really appreciate if images could be stored off in a remote resource and URL's are added as the image source. The repo size is increasing constantly and we like to keep it minimum.

Reply:
Done. Check now